src/domain/swap/index.ts

```diff
@@ -0,0 +1,24 @@
+export * from "./swap-out-checker"
+
+export const SwapProvider = {
+  Loop: "Loop",
+} as const
+
+export const SwapType = {
+  Swapout: "swapout",
+  Unknown: "unknown",
```

Comment:
What is this for?

Reply:
swapOut vs swapIn. We don't need swapIn's yet and the code is not written, so I did not yet include it in the SwapType

Reply:
hmm - then I guess this can be deleted. Don't want to keep dead code around.

Reply:
its actually used in the loop-service when parsing data from the loop grpc